content/03-reference/01-tools-and-interfaces/02-prisma-client/08-pagination/index.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: 'Pagination'
+metaTitle: 'Pagination (Reference)'
+metaDescription: 'Prisma Client provides features that are not achievable with relational databases. These features are referred to as "polyfills" and explained on this page.'
+---
+
+## Overview
+
+Prisma Client supports both offset and cursor-based pagination.
+
+## Offset pagination
+
+Offset pagination combines `skip` and `take` to skip a certain number of results and select a limited range. The following query skips the first 3 `Post` records and returns records 4 - 7:
+
+```ts line-number
+const results = await prisma.post.findMany({
+  skip: 3,
+  take: 4,
+})
+```
+
+![](pagination/offset-skip-take.png)
+
+### ✔ Pros of offset pagination
+
+- You can jump to any page immediately. For example, you can `skip` 200 records and `take` 10, which simulates jumping straight to page 21 of the result set. This is not possible with cursor-based pagination.
+- You can paginate the same result set in any sort order. For example, you can jump to page 21 of a list of `User` records sort by first name. This is not possible with cusor-based pagination, which requires sorting by a unique, sequential column.
+
+### ✘ Cons of offset pagination
+
+- Offset pagination **does not scale** at a database level. For example, if you skip 200,000 records and take the first 10, the database still has to traverse the first 200,000 records before returning the 10 that you asked for - this affects performance.
+
+### Use cases for offset pagination
+
+- Shallow pagination of results. For example, a blog interface that allows you to filter `Post` records by author and page through the first few pages.
+
+### Example: Filtering and offset pagination
+
+The following query returns all records where the `email` field contains `"prisma.io"`. The query skips the first 40 records and recods 41 - 51.
```

Comment:
quotes around search string intentional inside code block?

Reply:
No, not intentional - fixed, thanks!